client.go

```diff
@@ -15,6 +16,7 @@ type Client struct {
 	Stop               chan bool
 	subscribedChannels *subscribedChannels
 	binders            map[string]chan *Event
+	Errors             chan *error
```

Comment:
There's no benefit to using a pointer here. In fact doing so will have a negative performance impact (though an infinitesimally small one).